tests/remote_config/test_remote_configuration.py

```diff
@@ -301,7 +301,8 @@ def remote_config_asm_extra_services_available(data):
 
     @bug(library="dotnet", reason="APPSEC-11886")
```

Comment:
Would these changes fix the problems with .net?

Reply:
@daniel-romano-DD

Reply:
@robertomonteromiguel I believe that is unrelated to this PR, why don't we test that in a different one? https://github.com/DataDog/system-tests/pull/1782

Reply:
As far as I can tell, APPSEC-11886 describes exactly this issue, so I will update the PR.